examples/wave/wave-op-mpi.py

```diff
@@ -203,8 +222,21 @@ def main(ctx_factory, dim=2, order=3, visualize=False, lazy=False):
     else:
         local_mesh = mesh_dist.receive_mesh_part()
 
-    dcoll = DiscretizationCollection(actx, local_mesh, order=order,
-                    mpi_communicator=comm)
+    from meshmode.discretization.poly_element import \
+            QuadratureSimplexGroupFactory, \
+            default_simplex_group_factory
+    dcoll = DiscretizationCollection(
+        actx, local_mesh,
+        discr_tag_to_group_factory={
+            DISCR_TAG_BASE: default_simplex_group_factory(base_dim=dim, order=order),
+            DISCR_TAG_QUAD: QuadratureSimplexGroupFactory(2*order + 1),
```

Comment:
Add a comment justifying the quadrature order.

Reply:
https://github.com/inducer/grudge/pull/210/commits/d468071d582918d3f3c7b6962bd0168ac5560fb7